packages/wonder-stuff-core/src/index.js

```diff
@@ -1,11 +1,12 @@
 // @flow
-export {KindError} from "./kind-error.js";
+export {clone} from "./clone.js";
 export {Errors} from "./errors.js";
+export {errorsFromError, Order} from "./errors-from-error.js";
 export {getKindFromError} from "./get-kind-from-error.js";
 export {getOriginalStackFromError} from "./get-original-stack-from-error.js";
+export {KindError} from "./kind-error.js";
+export {safeHasOwnProperty} from "./safe-has-own-property.js";
```

Comment:
Added but I think it kinda negates the original issue. So I'll likely delete this from the core library.